tools/torch_quant/test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright 2023 The BladeDISC Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+pip install -r requirements-dev.txt
+python -m unittest discover -s tests
```

Comment:
Do we need to setup the package first?

Reply:
the quant frontend tool is pure python. no need to build package before testing.